docs/windows.md

```diff
@@ -421,59 +316,173 @@ Nodes and Pods in your cluster by running:
-control-plane                  Ready    control-plane,master   1h    v1.23.5   10.176.27.168   <none>        Ubuntu 20.04.3 LTS                        5.11.0-1022-generic   docker://20.10.14
-win-5akrf2tpq91                Ready    <none>                 1h    v1.23.5   10.176.27.150   <none>        Windows Server 2019 Standard Evaluation   10.0.17763.2686       docker://20.10.9
-win-5akrf2tpq92                Ready    <none>                 1h    v1.23.5   10.176.27.197   <none>        Windows Server 2019 Standard Evaluation   10.0.17763.2686       docker://20.10.9
+NAME                           STATUS   ROLES           AGE   VERSION   INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                         KERNEL-VERSION       CONTAINER-RUNTIME
+control-plane                  Ready    control-plane   1h    v1.29.0   10.176.27.168   <none>        Ubuntu 22.04.3 LTS               6.2.0-1017-generic   containerd://1.6.26
+win-5akrf2tpq91                Ready    <none>          1h    v1.29.0   10.176.27.150   <none>        Windows Server 2019 Datacenter   10.0.17763.5206      containerd://1.6.6
+win-5akrf2tpq92                Ready    <none>          1h    v1.29.0   10.176.27.197   <none>        Windows Server 2019 Datacenter   10.0.17763.5206      containerd://1.6.6
 
 # Show antrea-agent and kube-proxy Pods
 kubectl get pods -o wide -n kube-system | grep windows
 antrea-agent-windows-6hvkw                             1/1     Running     0          100s
 kube-proxy-windows-2d45w                               1/1     Running     0          102s
 ```
 
-### Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)
+### Installation as a Service
+
+Install Antrea (v0.13.0+ is required for containerd) as usual. The following
+command deploys Antrea with the version specified by `<TAG>`:
+
+```bash
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
+```
+
+When running the Antrea Agent as a Windows service, no DaemonSet is created for
+Windows worker Nodes. You will need to ensure that [nssm](https://nssm.cc/) is
+installed on all your Windows Nodes. `nssm` is a handy tool to manage services
+on Windows.
+
+To prepare your Windows worker Nodes, follow the steps in [Join Windows worker Nodes](#join-windows-worker-nodes).
+With this installation method, OVS daemons are always run as services (not
+containerized), and you will need to run `Install-OVS.ps1` to install them.
+
+When your Nodes are ready, run the following scripts to install the antrea-agent
+service. NOTE: `<KubernetesVersion>`, `<KubeconfigPath>` and
+`<KubeletKubeconfigPath>` should be set by you. `<KubeProxyKubeconfigPath>` is
+an optional parameter that is specific to kube-proxy mode. For example:
+
+```powershell
+# kube-proxy mode is no longer supported starting with K8s version 1.26
+$InstallKubeProxy=$false
+$KubernetesVersion="v1.23.5"
```

Comment:
Set `$KubernetesVersion` with version "1.29.0" to be consistent with the previous descriptions?